middleware/http.py

```diff
@@ -1,55 +0,0 @@
-from importlib import import_module
-
-from django.conf import settings
-from django.http import HttpResponseForbidden
-from django.template import RequestContext, Template, TemplateDoesNotExist, loader
-from django.utils.deprecation import MiddlewareMixin
-
-"""
-# Middleware to allow the display of a 403.html template when a
-# 403 error is raised.
-"""
-
-
-class Http403(Exception):
```

Comment:
This exception was never used, so whole middleware was useless, this is already handled by our `403.html` template